extension/lib/mergeCartsOnLogin.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const ShopgateCartPipeline = require('./shopgate/CartExtensionPipeline')
+
+module.exports = async (context, input) => {
```

Comment:
Input is not necessary.

Reply:
I know, but I'd rather stick to the "unrwritten convention" that a step module always take two arguments: context + input